.github/workflows/plugins.yml

```diff
@@ -264,7 +264,7 @@ jobs:
   express:
     runs-on: ubuntu-latest
     env:
-      PLUGINS: express
+      PLUGINS: express|body-parser
```

Comment:
Is this actually needed? How was body-parser tested before?

Reply:
I think we missed it when we splitted the tests